pom.xml

```diff
@@ -14,11 +14,11 @@
 
   <!-- Artifact Configuration -->
   <groupId>io.openliberty.arquillian</groupId>
-  <artifactId>arquillian-parent-liberty</artifactId>
-  <version>1.0.7-SNAPSHOT</version>
+  <artifactId>arquillian-parent-liberty-jakarta</artifactId>
+  <version>2.0.0-M1</version>
```

Comment:
I believe all these 2.0.0-M1 versions need to be 2.0.0-M1-SNAPSHOT. The process of doing the release build strips the SNAPSHOT off so that you get 2.0.0-M1. It then bumps the version to the next level SNAPSHOT for forward development.